torchvision/prototype/transforms/functional/_augment.py

```diff
@@ -7,16 +7,14 @@
 
 from ._utils import dispatch
 
-T = TypeVar("T", bound=features._Feature)
-
 
 @dispatch(
     {
         torch.Tensor: _F.erase,
         features.Image: K.erase_image,
     }
 )
-def erase(input: T, *args: Any, **kwargs: Any) -> T:
+def erase(input: Any, *args: Any, **kwargs: Any) -> Any:
```

Comment:
Is this because often the outcome in these kernels can be `torch.Tensor` and `PIL.Image`?

Reply:
Yes.